server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/pageobjects/issues/Issue.java

```diff
@@ -34,31 +32,31 @@ public Issue(SelenideElement elt) {
   }
 
   public Issue shouldAllowAssign() {
-    elt.find(".js-issue-assign").shouldBe(visible);
+    elt.find(".js-issue-assign").shouldBe(Condition.visible);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal ".js-issue-assign" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1192)